go/store/nbs/table_reader.go

```diff
@@ -75,14 +74,31 @@ func (cmp CompressedChunk) ToChunk() (chunks.Chunk, error) {
 	return chunks.NewChunk(data), nil
 }
 
+func ChunkToCompressedChunk(chunk chunks.Chunk) CompressedChunk {
+	compressed := snappy.Encode(nil, chunk.Data())
+	length := len(compressed)
+	compressed = append(compressed, []byte{0, 0, 0, 0}...)
+	binary.BigEndian.PutUint32(compressed[length:], crc(compressed[:length]))
+	return CompressedChunk{H: chunk.Hash(), FullCompressedChunk: compressed, CompressedData: compressed[:length]}
```

Comment:
@bheni Much open to suggestions on improving this and/or reducing any foreseeable problems in duplicating this from `table_writer`.

Reply:
So the TableWriter uses a snappyEncoder interface.  Sometimes this is a "realSnappyEncoder", but not necessarily.  Not sure how these play together.  I guess I'd leave the code like this, and just have CompressedChunks always use real compression.